docs(ai-agents): fix Context Store page bug bash items (#77736)
airbytehq/airbyte: docs/ai-agents/concepts/context-store.md

## How search works

> The Context Store supports structured search with filter operators, field selection, sorting, and cursor-based pagination. The agent translates user intent from natural language into a structured query, so most users don't need to construct queries by hand.

Author: ```suggestion
The Context Store supports structured search with filter operators, field selection, sorting, and cursor-based pagination. The agent translates user intent from natural language into a structured query, so you don't need to construct queries by hand.
```

## Limitations

- **Data freshness.** The Context Store is not real-time. Airbyte refreshes data on a schedule that depends on your plan, ranging from hourly to daily. See [Billing and pricing](../admin/billing) for refresh cadence by plan.

> - **Curated subset of data.** Not every field or entity from a source is included. Airbyte indexes only the fields and entities that are most useful for search. For the list of indexed entities per connector, see [Agent connectors](../connectors).

Author: We actually don't document this. However, when selecting entities in the authentication widget, you can see which entities populate the Context Store.

> - **Curated subset of data.** Not every field or entity from a source is included. Airbyte indexes only the fields and entities that are most useful for search. You can see which entities populate the Context Store when selecting entities in the connector authentication widget.

Author: ```suggestion
- **Curated subset of data.** Not every field or entity from a source is included. Airbyte indexes only the fields and entities that are most useful for search. You can see which entities populate the Context Store when selecting entities in the authentication widget.
```
